index.html

```diff
@@ -23,6 +23,13 @@ <h1>Lambda School Newsfeed</h1>
       </ul>
     </div>
     <div class="articles">
+        <div class="article">
+            <h2>John Spraul for President</h2>
+            <p class="date">Nov 5th, 2017</p>
+      
+            <p>'Nuff said.</p>
```

Comment:
don't think that this will distract me from your dropdown menu not working properly 👀